<?php

namespace ContainerIafxDCP;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_XNeJFoService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.X_neJFo' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.X_neJFo'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'requestDto' => ['privates', 'App\\Api\\Product\\Dto\\ValidationExampleRequestDto', 'getValidationExampleRequestDto2Service', true],
            'validationErrors' => ['privates', '.errored..service_locator.X_neJFo.Symfony\\Component\\Validator\\ConstraintViolationListInterface', NULL, 'Cannot autowire service ".service_locator.X_neJFo": it references interface "Symfony\\Component\\Validator\\ConstraintViolationListInterface" but no such service exists. Did you create a class that implements this interface?'],
        ], [
            'requestDto' => 'App\\Api\\Product\\Dto\\ValidationExampleRequestDto',
            'validationErrors' => 'Symfony\\Component\\Validator\\ConstraintViolationListInterface',
        ]);
    }
}
